### PR TITLE
Db setup and entity creation

### PR DESCRIPTION
**Pull request by nomss**
Db setup and entity creation.
Then creating service methods for getting the User and updating the user.

### CODE REVIEW

File: bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping()
+    public UserDTO createUser(@RequestBody UserRequest userRequest) {
+        return userService.createUser(userRequest.username(), userRequest.password());
+    }
+
+    @GetMapping("/getUser")
```
Comment (tarikmujcic) [inline bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java]: This is wrong.

1. @GetMapping shouldn't be /getUser it is already a get request so no need for get and we already have users in api/v1/users so no need to add getUser.
This is how it should be:
@GetMapping("/{id}")

2. Method name is wrong. It should not be create user it should be getUser.

This is how it should look like:

```
@GetMapping("/{id}")
public UserDTO createUser(@PathVariable Long id) {
        return userService.getUser(1L);
    }
```

File: bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.ayeeti.blservice.user;
+
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/users")
+public class UserController {
```
Comment (tarikmujcic) [inline bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java:9]: You are missing DELETE and PUT endpoints here.

**Review by tarikmujcic — CHANGES_REQUESTED** [review]

File: bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java (hunk 2 above)
Comment (nomss) [reply bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java:9]: Done.

File: bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java (hunk 1 above)
Comment (nomss) [reply bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java]: Done.

**Pull request closed without merging**